inc/namespace.php

```diff
@@ -46,6 +48,10 @@ function bootstrap() {
 
 	$is_alb_healthcheck = isset( $_SERVER['HTTP_USER_AGENT'] ) && strpos( $_SERVER['HTTP_USER_AGENT'], 'ELB-HealthChecker' ) === 0;
 
+	require_once __DIR__ . '/fluent_bit/namespace.php';
+	require_once __DIR__ . '/fluent_bit/error_handler/namespace.php';
```

Comment:
Include these via the `composer.json` autoload section. If you're having issues with the autoload.php not including them then try removing `composer.lock` and `vendor/altis/cloud` then running `composer update`.

Reply:
Aha! Thank for you noting how to update dependencies, that was helpful!